Libraries/LibC/sys/time.h

```diff
@@ -64,6 +42,95 @@ struct timezone {
     int tz_dsttime;
 };
 
+
 int gettimeofday(struct timeval* __restrict__, void* __restrict__) __attribute__((nonnull(1)));
 
+
+static inline void timeradd(struct timeval* a, struct timeval* b, struct timeval* out)
```

Comment:
```suggestion
static inline void timeradd(const struct timeval* a, const struct timeval* b, struct timeval* out)
```

Reply:
Err, good point. https://man7.org/linux/man-pages/man3/timercmp.3.html didn't have them and I wondered about it, but that page says "macros" so anything goes. Changing, thanks!